jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -203,7 +217,22 @@ export class MountPlugin implements IMountPlugin {
     );
     this._datum.addClass('pachyderm-mount-datum-wrapper');
 
+    this._pipeline = ReactWidget.create(
```

Comment:
We touched upon this during the FE walkthrough with @pleeko in Miami, but we probably don't need to use signals here since the pipeline component doesn't have any properties/data that changes like the other components does. Signals are useful in the case where we see that some data has changed so we re-render the components that depend on that data. But since the pipeline component doesn't take any input that could change, we can remove signals.

Now, how this would look with no signals, I defer to Peter.

Reply:
I removed the signal in a way that made sense to me, but I'll wait for Peter's feedback